model/license.go

```diff
@@ -302,6 +307,13 @@ func (l *License) IsSanctionedTrial() bool {
 		(duration >= sanctionedTrialDurationLowerBound.Milliseconds() || duration <= sanctionedTrialDurationUpperBound.Milliseconds())
 }
 
+func (l *License) HasEnterprisePluginFeatures() bool {
+	return *l.Features.EnterprisePlugins ||
+		l.SkuShortName == LicenseShortSkuE20 ||
+		l.SkuShortName == LicenseShortSkuProfessional ||
```

Comment:
Just to confirm, is "professional' an Enterprise SKU (that is, belonging to the 'Enterprise bucket)?

Reply:
@catalintomai In general Professional is not considered an Enterprise SKU, but Professional is above E10 in that it should have no restrictions for existing plugin features, including downloading the plugins from the marketplace. However, new individual features of plugins may discriminate by SKU on a granular level.

Please see https://community-daily.mattermost.com/core/pl/q6zkyz4hu3y1td1zfqchoca1ye for the most recent discussion on which SKUs should have access to installing enterprise plugins.

Reply:
You can also take a look at this doc for feature availability per license tiers https://docs.google.com/document/d/1nDO4NZwwxSNm_h-meJXitnNajETQ2rPam1X4D6bwfDo/edit#